app/assets/stylesheets/shared/tabs.scss

```diff
@@ -1,146 +1,75 @@
-.sidetabs {
-  position: relative;
-  background: image-url("prx-mark-outlined.svg") repeat top left $gray-100;
-  padding: 0;
-  margin: 0;
-  display: flex;
-  flex-direction: row;
-
-  @media (min-width: 768px) {
-    flex-direction: column;
-  }
-
-  .list-group {
-    border-bottom: none;
-    position: relative;
-    display: flex;
-    flex-direction: row;
+:root {
+  --prx-tab-bg-rgb: #{to-rgb($gray-200)};
+  --prx-tab-bg-opacity: 1;
+  --prx-tabs-gap: 1px;
+}
```

Comment:
I overlooked the tabs yesterday, I'd like to see a couple of changes. 

The `--prx-tabs-gap` variable is being overridden by `.gap-1` which is .25rem and too big. Honestly, I only want to see a 1px gap between tabs. I

There's also like 3-5 pixels worth of extra gap or padding at the end of the last tab. Can you remove that? 

Thanks

Reply:
<img width="242" alt="Screen Shot 2023-02-09 at 10 52 30 AM" src="https://user-images.githubusercontent.com/369739/217892517-a7fa926c-4040-41df-a67a-5e4936dd1676.png">